### ISSUE TITLE
UX -  Plugin Setting page should have Notification/Message when changing/updating any setting 

### ISSUE THREAD

**Issue by ankitguptaindia** [buddyboss/buddyboss-platform#660]
**Describe the bug**
As a UX point of view, When we change/update any Platform plugin setting, it should show a message like `Setting Saved Successfully` 

**To Reproduce**
1. Go to BuddyBoss Tab and then Setting Page and change/update any setting.
2. Click on Save Setting

**Expected behavior**
Should show Setting Saved message in Header. Same as when we save Redux setting in Theme.

**Screenshots**
Available in theme but not in Plugin https://prnt.sc/rj1ejk
https://www.loom.com/share/2f1f6fa1e806462e9c250e16f486f5b7

**Support ticket links**
N/A - Issue from QA

**Jira issue** : [PROD-953]

[PROD-953]: https://buddyboss.atlassian.net/browse/PROD-953?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
